### ISSUE TITLE
Problem loading an extension

### ISSUE THREAD

**Issue by lucasreisprestes** [rainbowcreatures/FlashyWrappers#49]
Hello,

I'm getting this message when trying to run the example "**The content cannot be loaded because there was a problem loading an extension: Error: Requested extension com.rainbowcreatures.FWEncoderANE is not supported for Windows-x86.**". 

I tested with  SDK 27.0.0.132, SDK 28.0.0.125 and I could not execute. I'm using Windows 10 64x and Adobe Animate CC 2017.

Any idea how to solve this?

**Comment by lucasreisprestes** [comment]
Any idea how to solve this? Thank you!

**Comment by rainbowcreatures** [comment]
Not really, I've recently updated the ANE to fix a bug for Windows so I know that extension is running fine on Win (also got Win 10 64x + Animate). 

Make sure you're including the right ANE (for Windows, not OSX, iOS etc.) and that its also typed out in  the app's xml, in the extensions tag (I believe).
